lib/little-loader.js

```diff
@@ -51,27 +55,30 @@
    * @param {Object}    context   (Optional) Callback context
    * @returns {void}
    */
-  window._lload = function (src, callback, context) {
+  var _lload = function (src, callback, context) {
     /*eslint max-statements: [2, 20]*/
     context = context || this;
```

Comment:
Wooooah! I didn't know about this!

```
function f(a){
  "use strict";
  a = 42;
  return [a, arguments[0]];
}
var pair = f(17);
console.assert(pair[0] === 42);
console.assert(pair[1] === 17);
```

So yes, adding `use strict` would make arg reassignment a safe operation. Thanks for showing me that!!! However, It's less explicit and if the perf team gets ahold of this code the'll strip the `use strict` statement ( :trollface: ).